lib/ecto_crux.ex

```diff
@@ -537,6 +534,14 @@ defmodule EctoCrux do
       defp to_map(list) when is_list(list), do: list |> Enum.into(%{})
       defp to_map(map) when is_map(map), do: map
 
+      def page_to_offset(page, page_size) when is_integer(page) and is_integer(page_size),
+        do: page_size * (page - 1)
+
+      def offset_to_page(0, page_size) when is_integer(page_size), do: 1
+
+      def offset_to_page(offset, page_size) when is_integer(offset) and is_integer(page_size),
+        do: (offset / page_size) |> Float.ceil() |> round()
```

Comment:
I think you have an edge case issue.
e.g if `offset = 30`, and `page_size = 30`, you should get `page = 2` (to be coherent with `page_to_offset`).
I'd transform it to `(offset / page_size) + 1 |> Float.floor() |> round()`

Reply:
The formula needs to be different from `crux_total_pages` since this is an offset, not a number of entries (`offset = 0` -> `entry = 1`)

Reply:
You are completely right i messed up when creating it